### PR TITLE
Add frontmatter to exporting connectors

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

This PR adds some frontmatter to the existing exporting engine connector READMEs so that they will display better and have better SEO on Learn.

##### Component Name

exporting

### CODE REVIEW

**Comment by joelhans** [comment]
@vlvkobal Who else do you think could review this? I'm not sure who else is backing you up on this project.

File: exporting/README.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 <!--
----
-title: "Exporting metrics to external databases (experimental)"
+title: "Export metrics to external databases (experimental)"
```
Comment (thiagoftsm) [inline exporting/README.md:2]: @vlvkobal will we keep the `experimental` here?

Comment (vlvkobal) [reply exporting/README.md:2]: It will be changed in another PR. I'm working on the document right now.

Comment (thiagoftsm) [reply exporting/README.md:2]: Thank you!

**Pull request merged**